### ISSUE TITLE
Open custom vocab

### ISSUE THREAD

**Issue by Daniel-KM** [omeka-s-modules/CustomVocab#38]
In some cases, librarians want to be able to complete an authority list when filling a resource. So i implemented it via the module AdvancedResourceTemplate. 

![image](https://user-images.githubusercontent.com/2138485/234516545-8fea2488-bbdd-49a6-a0c9-7fd7abfc901c.png)

Maybe i can move code here. It requires a new column in database (open/closed list) and some js. The main point is to define rights to complete a custom vocab.

**Comment by jimsafley** [comment]
Thanks for the suggestion and the offer. I'm sure the feature is helpful for some use cases, but it is rarely good practice to allow adjustments to a controlled vocabulary during data entry. It's a good idea to make it optional.

We're open to a PR, but remain skeptical about the feature. Like always, we're more open to accept a PR if it's self-contained and employs the minimum amount of code needed to make the feature work.

**Comment by Daniel-KM** [comment]
Yes, it's a bad practice that can be replaced by another feature in AdvancedResourceTemplate that allows autocompletion from the current list of values, but for a project, they want a custom vocab.

 ![autocompletion](https://github.com/Daniel-KM/Omeka-S-module-AdvancedResourceTemplate/blob/master/data/images/autocompletion.png?raw=true)

So i keep it in the module for now, it may be a future pr.

**Comment by jimsafley** [comment]
> another feature in AdvancedResourceTemplate that allows autocompletion from the current list of values

The ValueSuggest module introduced that feature in [v1.14.0](https://github.com/omeka-s-modules/ValueSuggest/releases/tag/v1.14.0). The feature seems more aligned with the purpose of ValueSuggest than CustomVocab.
